### PR TITLE
Testing part 1

### PR DESCRIPTION
**Pull request by MAleemH**
> Created a test file named todo.test.js. Added add item and delete item functions for testing.

> It test add/delete functions add or remove exactly one li element from the list in the DOM.

> We have grouped our tests using the describe() method.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,170 @@
...
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
```
Comment (GzimAsani) [inline README.md]: - [ ] Please remove descriptions from your readme file
image

File: README.md [hunk 2]
```diff
@@ -1 +1,170 @@
...
+
+```sh
+  npm run build
+  npm run start
+```
+
+### Run tests
+
+After the above step, program will start running. You can perform desired activities.
+
```
Comment (GzimAsani) [inline README.md]: - [ ] Explainto potential users the command they need to write to run tests 
image

File: README.md [hunk 3]
```diff
@@ -1 +1,170 @@
...
+👤 **Shakir Hussain**
+
+- GitHub: [@Shakir-Hussain12](https://github.com/Shakir-Hussain12)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
```
Comment (GzimAsani) [inline README.md:129]: - [ ] Please add 1-3 potential future features 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @MAleemH  and @Shakir-Hussain12                              :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your tests, they are all passing  :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**